DragonLoopApp/DragonLoopApp.iOS/CustomRenderer/CustomRenderer.cs

```diff
@@ -35,7 +35,7 @@ protected override void OnElementChanged(ElementChangedEventArgs<View> e)
             {
                 var formsMap = (CustomMap)e.NewElement;
                 var nativeMap = Control as MKMapView;
-                customPins = formsMap.CustomPins;
+                customPins = formsMap.CustomPins ?? customPins;
```

Comment:
Similar comment, is this still needed with the changes to the CustomMap constructor?